gammapy/modeling/models/temporal.py

```diff
@@ -592,13 +592,16 @@ def read(cls, filename, format="table"):
             with fits.open(filename) as hdulist:
                 header = hdulist["SKYMAP_BANDS"].header
                 t_ref = time_ref_from_dict(header)
-                # TODO : Ugly hack to prevent creating a TimeMapAxis
-                # By default, MapAxis.from_table tries to create a
-                # TimeMapAxis, failing which, it creates a normal MapAxis.
-                # This ugly hack forces the fail. We need a normal Axis to
-                # have the evaluate method work
-                hdulist["SKYMAP_BANDS"].header.pop("MJDREFI")
-                m = RegionNDMap.from_hdulist(hdulist)
+                m1 = RegionNDMap.from_hdulist(hdulist)
+                axes_new = []
+                for ax in m1.geom.axes:
+                    if isinstance(ax, TimeMapAxis):
```

Comment:
From a first look I don't understand why this works. I thought the energy dependent light-curve format would store the center bins of the time. So how can this be parsed as a `TimeMapAxis` before and then replaced later? What have I missed?